tests/utils.py

```diff
@@ -138,6 +139,7 @@ async def async_wait_until(fn, timeout=5000, interval=100):
         Waiting interval, by default 100
 
     Adapted from pytest-qt.
+
```

Comment:
I don't understand this empty line, as other docstrings do not necessarily have it. But if this is what ruff say, so be it.